src/substrate-lib/substrateContext.tsx

```diff
@@ -111,9 +111,9 @@ const loadAccounts = (state, dispatch) => {
   asyncLoadAccounts();
 };
 
-const SubstrateContext = React.createContext();
+const SubstrateContext = React.createContext(undefined);
```

Comment:
Does the compiler complain if you leave out the `undefined` parameter?

Reply:
Yes, `React.createContext()` requires 1 parameter (a default value).